src/main/java/com/shipmentEvents/handlers/EventHandler.java

```diff
@@ -52,6 +52,7 @@ public String handleRequest(ScheduledEvent scheduledEvent, Context context) {
             return "SUCCESS";
         } catch (final Exception ex) {
             logger.log(String.format("Failed to process shipment Updates in %s due to %s", scheduledEvent.getAccount(), ex.getMessage()));
+            tgdsniogudfniogdf
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;;&#34; to complete LocalVariableDeclarationStatement